wallet/src/de/schildbach/wallet/ui/staking/StakingActivity.kt

```diff
@@ -60,10 +60,12 @@ class StakingActivity : LockScreenActivity() {
         super.onCreate(savedInstanceState)
 
         binding = ActivityStakingBinding.inflate(layoutInflater)
-        lifecycleScope.launch { navController = setNavigationGraph() }
+        lifecycleScope.launch {
+            navController = setNavigationGraph()
+            viewModel.observeOnlineAccountStatus().observe(this@StakingActivity, ::handleOnlineAccountStatus)
+        }
 
         viewModel.navigationCallback.observe(this, ::handleNavigationRequest)
-        viewModel.observeOnlineAccountStatus().observe(this, ::handleOnlineAccountStatus)
```

Comment:
there was a race condition.  if the observer ran before the `navController` was initialized, then it would crash.  This was a rare crash.